### ISSUE TITLE
UNIQUE constraint not working in sqlite3 connector

### ISSUE THREAD

**Issue by Coteh** [eveningkid/denodb#18]
I create a user table using this model:
```
class User extends Model {
  static table = "users";
  static timestamps = true;

  static fields = {
    id: {
      primaryKey: true,
      autoIncrement: true,
    },
    name: DATA_TYPES.STRING,
    email: {
        type: DATA_TYPES.STRING,
        unique: true,
        allowNull: false,
        length: 50
      },
  };
}
```
The structure looks like this when opened in a DB viewer: (I use TablePlus on macOS)
```
CREATE TABLE `users` (`id` integer not null primary key autoincrement, `name` varchar(255), `email` varchar(50) not null, `created_at` datetime not null default CURRENT_TIMESTAMP, `updated_at` datetime not null default CURRENT_TIMESTAMP);
```
There doesn't seem to be a `UNIQUE` constraint for the `email` column, so I'm able to insert multiple records with same email.
I am able to report this issue for the `sqlite3` connector, not sure if it exists in the other two yet.

**Comment by Coteh** [comment]
I think I figured out the issue:
https://github.com/denjucks/dex/blob/a3db9a4f3bfba8f81d7134557d05ac7821e090be/lib/schema/tablebuilder.js#L51-L55
You need to specify the column name for the `primary` and `unique` methods. I will make a PR in a bit.

**Comment by Coteh** [comment]
Weird, I saw on Chrome debugger that the `ALTER` commands were being applied to the columns to add the uniqueness constraint, but does not appear in the resulting sqlite table 🤔

**Comment by eveningkid** [comment]
Interesting. Good this can be fixed with #19. I'll wait on you so we can close this issue.

Thank you James!

**Comment by Coteh** [comment]
Fixed with #19.
